app/femr/business/services/system/SearchService.java

```diff
@@ -423,6 +423,7 @@ public SearchService(IRepository<IDiagnosis> diagnosisRepository,
         String firstName = null;
         String lastName = null;
         String firstOrLastName = null;
+        Integer phoneNo = null;
         if (words.length == 0) {
```

Comment:
The phone numbers are stores as a string and really should be queried via string matching. Look into using `like` with wildcards to match the phone numbers. The SQL query should probably end up looking like:

```
... where phone_no LIKE '%1234%'  ...
```
This search function is a good place to start, but I think FEMR-147 is specifically referencing the search box on the home page. You'll need to look into the javascript used for the home page search as well.


Also, this would be best to submit in its own pull request since it is a different issue.